x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/dataframe/action/GetDataFrameTransformsStatsAction.java

```diff
@@ -55,6 +55,7 @@ public Response newResponse() {
     public static class Request extends BaseTasksRequest<Request> {
         private final String id;
         private PageParams pageParams = PageParams.defaultParams();
+        private boolean allowNoResources = true;
```

Comment:
Rename `allowNoResources` to `allowNoMatch` throughout this file, including the getter and setter.